queries/ebnf/highlights.scm

```diff
@@ -0,0 +1,47 @@
+;;;; Simple tokens ;;;;
+(terminal) @string @grammar.terminal
```

Comment:
CI fails because `@grammar` is a undocumented new highlight type, There are a few possibilities to fix this:

- Is `@grammar` needed for other languages? Then `@grammar` or better `@grammar.punctuation` should be documented in `CONTRIBUTING.md` and added to all languages where necessary
- Is `@grammar` similar to a existing highlight group in `CONTRIBUTING.md`? Then, it should be changed to `@XXX.grammar.nonterminal` where `XXXX` is a documented capture. This way users will have a good default experience when their color scheme does not support `@grammar`

Reply:
Yes, that would be ok. But I think your old `@grammar` categories were also ok, since they can be surely extended to other languages

Reply:
The `@grammar` captures could also be used for ANTLR/Yacc/Bison/JFlex/CUP/etc. but none of them have tree-sitter grammars (that I know of) yet and I think existing captures can adequately describe all their tokens anyway.

Highlights can currently be customised with `@string.ebnf` and so on without needing a custom suffix.